test/openAPI/features/client_create.feature

```diff
@@ -1,16 +1,130 @@
-Feature: API to add new open ID connect (OIDC) clients, it can be invoked by other modules which manage the relying parties/partners.
+@method=POST @endpoint=/client-mgmt/oidc-client
+Feature: API to add new open ID connect (OIDC) clients.
```

Comment:
This PR is about OIDC update oidc client APIs ? I see endpoint, file namings and everything as create here.

Reply:
Yes, it's true. But there is also client_create file because I needed it for client_update so I create a new branch from https://github.com/GovStackWorkingGroup/bb-identity/pull/27 branch ;)
So please add review only for client_update ;)

Reply:
ok